Cargo.toml

```diff
@@ -18,9 +18,10 @@ exclude = [
 ]
 version = "1.0.0-rc.3"
 edition = "2021"
-# We need at least 1.65 for GATs
-# https://blog.rust-lang.org/2022/11/03/Rust-1.65.0.html
-rust-version = "1.65"
+# We need at least 1.65 for GATs[1] and 1.67 for `ilog`[2]
+# [1] https://blog.rust-lang.org/2022/11/03/Rust-1.65.0.html
+# [2] https://blog.rust-lang.org/2023/01/26/Rust-1.67.0.html#stabilized-apis
+rust-version = "1.67"
```

Comment:
🗺️ Small bump to the minimum Rust version to enable us to use the stabilized `ilog10` method for counting decimal digits in integers.